store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9802,4 +9802,24 @@ TODO: delete them permanently from here
     <defaultCOSValue>TRUE</defaultCOSValue>
     <desc>Whether Powerpaste preference is enabled by user or admin</desc>
 </attr>
+<attr id="3086" name="zimbraPrefImapEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.0.0">
```

Comment:
9.1.0

Reply:
Next release version is yet to be finalised. Keeping 9.0.0